canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx

```diff
@@ -216,6 +217,33 @@ export default class AbstractTable extends React.Component {
 		}
 	}
 
+	/**
+	* Returns array of visible values in second column
+	*/
+	getCheckboxLabels() {
+		const controlValue = this.props.value ? this.props.value : [];
+		const labels = [];
+		for (let i = 0; i < controlValue.length; i++) {
```

Comment:
Have you tested this with nested tables, where the second column is a structure? This may need to be recursive